platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -451,6 +451,8 @@ sep38:
   # Whether to enable SEP-38
   #
   enabled: false
+  # Whether to require SEP-10 authentication for SEP-38 requests.
+  requires_sep10: false
```

Comment:
SEP-38 has other endpoints that require authentication such as `POST /quote`. Should we change the name of this flag or schema (nesting it under `sep38.endpoints.info.authentication_required`, for example) so that this is clear?